index.js

```diff
@@ -72,6 +72,10 @@ module.exports = {
                     'justify-self',
                 ],
             },
+            {
+                // Align + Justify.
+                properties: ['place-items'],
+            },
```

Comment:
In this config the shorthand properties are ordered before the more specific ones. Could you combine the "align" and "justify" groups into a single one called "Layout alignment." that has `place-items` first, followed by the `align-` properties, and then the `justify-` ones?

Reply:
Sure thing.